main.js

```diff
@@ -0,0 +1,64 @@
+// 1. Написати функцію, яка приймає 4 аргументи і шукає найменше число серед них
+const GET_MIN_NUM = function (num1, num2, num3, num4) {
+    const MIN = Math.min(num1, num2, num3, num4);
+}
+GET_MIN_NUM()
+
+//2. Написати функцію, яка приймає 2 аргументи та шукає найбільший спільний дільник для них. Якщо такого числа немає, то повідомляти про відсутність
+const GET_GREATEST_COMMON_DENOMINATOR = function (num1, num2) {
+    if ((typeof num1 !== 'number') || (typeof num2 !== 'number')) 
+      return false;
+      num1 = Math.abs(num1);
+      num2 = Math.abs(num2);
+    while(num2) {
+      const NUM3 = num2;
+      num2 = num1 % num2;
+      num1 = NUM3;
+    }
+    return num1;
+  }
+GET_GREATEST_COMMON_DENOMINATOR()
+
+// 3. Написати функцію, яка приймає 1 аргумент та перевіряє чи дане число ідеальне
+const PERFECT_NUMBER = function (num) {
+    let step = 0;
+    for (let i = 1; i <= num/2; i++) {
+        if (num % i === 0) {
+            step += i;
+        }
+    }
+    if (step === num && step !== 0) {
+        //console.log(`The number ${num} is perfect!`)
+    } else {
+        //console.log(`The number ${num} is NOT perfect!`)
+    }
+}
+PERFECT_NUMBER()
```

Comment:
![Screenshot 2022-09-19 at 22 08 22](https://user-images.githubusercontent.com/34886027/191096263-3a70c546-0290-437d-a565-895d6ee2a1f0.png)

Reply:
And not other cases covered correctly